### DISCUSSION TITLE
the so-preflight script failed checking one or more URLs required by setup

### DISCUSSION THREAD

**Discussion by o-zero-o** [Security-Onion-Solutions/securityonion#10658] [General]
When internet access is available, error "the so highlight script failed checking one or more URLs required by setup" is reported.

It should be noted that I am a Chinese user and can ping through ghrc.io or github during testing. But I can't ping google.com

**Comment by cm-ops** [comment]
Are you able to get to all the appropriate sites listed here? https://docs.securityonion.net/en/latest/firewall.html#internet-communication

**Reply by o-zero-o** [reply]
The first installation can be successfully deployed and  web page can be opened. But later, the service inexplicably went offline, and after restarting the server, all services could not start automatically. After restoring the snapshot, the downloaded docker inexplicably disappeared and most be reinstalled. In this installation, all configurations are identical to the initial installation, but individual URLs are no longer accessible.
